src/db-up/azext_db_up/util.py

```diff
@@ -30,8 +30,12 @@ def create_random_resource_name(prefix='azure', length=15):
 def get_config_value(db_type, option, fallback='_fallback_none'):
     config_section = CONFIG_MAP[db_type]
     if fallback == '_fallback_none':
-        return DB_CONFIG.get(config_section, option)
-    return DB_CONFIG.get(config_section, option, fallback=fallback)
+        result = DB_CONFIG.get(config_section, option)
+    else:
+        result = DB_CONFIG.get(config_section, option, fallback=fallback)
+    if result:
+        return result
+    return None
```

Comment:
If the `result` is `0` or `False`, the result of function returns None, is that as expected?

Reply:
yes